src/scripts/site.js

```diff
@@ -14,7 +14,7 @@ function getWindowLocationSearch(win) {
     return search || '';
 }
 
-function getParameterByName(name, url) {
+window.getParameterByName = function (name, url) {
```

Comment:
why add function explicitly?

Reply:
It's tripping up ESLint, as it thinks it's unused.